src/components/logo/logo.tsx

```diff
@@ -30,11 +32,40 @@ const mapStateToProps = state => ({
 @withLogger(COMPONENT_NAME)
 @withText({logoText: 'controls.logo'})
 class Logo extends Component<any, any> {
+  private _logoRef: RefObject<HTMLAnchorElement> = createRef<HTMLAnchorElement>();
+
+  /**
+   * when component did mount
+   *
+   * @returns {void}
+   * @memberof Logo
+   */
+  public componentDidMount(): void {
+    this._handleLogoUrl();
+  }
+
+  /**
+   * handles the logo url
+   * if the url contains ${entryId}, then replace it with the played entry id
+   *
+   * @returns {void}
+   * @memberof Logo
+   */
+  private _handleLogoUrl(): void {
+    const url = this.props.config.url;
+    if (url.indexOf(ENTRY_VAR) !== -1) {
+      const {player} = this.props;
+      player.addEventListener(player.Event.CHANGE_SOURCE_ENDED, () => {
```

Comment:
you should use eventManager for this 
otherwise you have to unregister to event on onUnMount lifesycle